### PR TITLE
Billing: Hide Item's valid from/to fixed if not enabled in BillingBas…

### PR DESCRIPTION
**Pull request by NinoRy**
…e global config, check boxes by default in every case

### CODE REVIEW

**Comment by NinoRy** [comment]
This needs to be tested a while, because small but critical changes were made in Generic BaseViewController!!
@Reviewer: Please strongly check added help messages in terms of understandability!!

File: resources/lang/de/helper.php [hunk 1]
```diff
@@ -32,8 +33,8 @@
 	//ItemController
 	'Item_ProductId'				=> 'Alle Felder außer dem Abrechnungszyklus müssen vor eine Änderung des Produkts gelöscht werde! Andernfalls können die Produkte in den meisten Fällen nicht gespeichert werden.',
 	'Item_ValidFrom'				=> 'Für einmalige (Zusatz-)Zahlungen kann das Feld genutzt werden, um die Zahlung zu teilen - nur  Jahr und Monat (jjjj.mm) werden berücksichtigt.',
-	'Item_ValidFromFixed'			=> 'Feste Termine werden für die Abrechnung verwendet und werden nicht durch externe Aufträge aktualisiert.',
-	'Item_ValidToFixed'				=> 'Feste Termine werden für die Abrechnung verwendet und werden nicht durch externe Aufträge aktualisiert.',
+	'Item_ValidFromFixed'			=> 'Dieses Feld ist standardmäßig gesetzt! Deaktivieren Sie diese Checkbox, wenn das Gültig ab - Datum ungewiss ist. In dem Fall wird bei Erreichen des Datums dieses jeden Tag erneut auf den darauffolgenden Tag gesetzt. Der Tarif startet damit nicht und wird auch nicht abgerechnet bis die Checkbox wieder aktiviert wird. Feste Termine werden nicht durch externe Aufträge (z.B. vom Telefonie-Provider) aktualisiert.',
```
Comment (schmto) [inline resources/lang/de/helper.php]: Quit complex :) for me. Can you re-discuss with Rene

Comment (schmto) [reply resources/lang/de/helper.php]: Maybe describing the practical use-cases is more efficient? I don't know

Comment (schmto) [reply resources/lang/de/helper.php]: And: Gültig ab - Datum würde ich in Hochkomme oder so schreiben, sonst könnte man denken es ist ein Gedankenstrich

**Comment by schmto** [comment]
I guess the modified Controllers are tested well

**Pull request merged**